pt-br/step_by_step/README.md

```diff
@@ -0,0 +1,44 @@
+# Passo a passo
+
+Antes de entrarmos em mais detalhes, vamos dar uma olhada no quadro geral da organização de um evento Django Girls.
+
+Você pode começar oganizando com 3 meses de antecedência. Isso dará uma quantidade de tempo segura para fazer tudo correr bem. Claro, a quantidade total de trabalho se encaixa entre 40-50 horas, por isso, se você dividir em 3 meses, você consegue fazer isso no seu tempo livre.
+
+Vamos ver como fica isso na linha do tempo:
+
+### 3 meses antes
+
+- __Inscreva-se na organização das Django Girls__
+- Faça um [website](website/README.html), [Twitter or Facebook](promotion/README.html)

```

Comment:
Be aware that all links to "`README.html`" will be broken! See Pull Request #19 for details.

You should rename the links to "`README.md`", or remove the document names in the links alltogether. You may wait to see what happens with #19 before touching your files.